app/views/invitations/new.html.haml

```diff
@@ -0,0 +1,19 @@
+%section.invite.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for @invitation, class: "invite" do |f|
+        %header
+          %h1 Invite a friend to join MyFlix!
+        %fieldset
+          .form-group
+            .row
+              .col-sm-4
+                = f.text_field :recipient_name, label: "Friend's Name"
```

Comment:
Since you are using `bootstrap_form_for` you don't have to use this:

```haml
.form-group
  .row
    .col-sm-4
```

Homework: Read bootstrap form gem docs and improve this form. Refactor to better code.